neurolib/models/wc/timeIntegration.py

```diff
@@ -232,7 +232,7 @@ def S_I(x):
                     * S_I(
                         c_excinh * excs[no, i - 1]  # input from the excitatory population
                         - c_inhinh * inhs[no, i - 1]  # input from within the inhibitory population
-                        + exc_input_d[no]  # input from other nodes
+                        + inh_input_d[no]  # input from other nodes
```

Comment:
I believe there is a bug in neurolib's native Wilson-Cowan so I fixed it, can @ChristophMetzner confirm? (excitatory input / coupling was given to inhibitory mass)

Reply:
Wait, that is supposed to be excitatory input, because there is no inhibition between different nodes. However, we can discuss whether the inhibitory population should get any input from other nodes or if it should only be excitatory to excitatory coupling between nodes.

Reply:
Ah, so the model is expected to have excitatory coupling at the network level also in inhibitory mass? then I am sorry, I thought it's a bug:) I'll remove then

Reply:
fixed (I mean taken back)... in light of this I also added excitatory -> inhibitory coupling to multimodel's implementation :)

Reply:
I guess most models only have excitatory-excitatory coupling at the network level, but excitatory-inhibitory coupling also exists in biology, so it does make some sense to have this coupling. However, I am also fine with only having excitatory-excitatory coupling in the WC to keep it consistent with all other models.

Reply:
I totally agree - there are exc->inh coupling in the biology and e.g. I am using exc-> inh at the network level to the thalamus... I left native WC in `neurolib` as is (that is **including** the coupling) and for multimodel implementation, I added it as well, but it can be "turned off" with one line, so you can use both versions and even compare them etc.

Reply:
Awesome!